docs/templating/design_engine/design_engine.md

```diff
@@ -27,25 +27,38 @@ ibexa:
 
 Each scope can use only one design.
 
+## Design theme list
+
+A theme is a set of directories to look for templates in. Mainly, theme's templates are placed in a directory under `templates/themes` which name is the theme name.
```

Comment:
```suggestion
A theme is a set of directories to look for templates in. A theme's templates are placed in a directory under `templates/themes` which has the same name as the theme.
```
I've removed "mainly", because the directory _must_ be named the same as the theme, AFAIK?

Reply:
@DominikaK 
I have a "mainly" because a theme have more paths than the templates/themes/ one as seen in the next section.
Inspired by https://github.com/ibexa/design-engine/blob/main/src/bundle/DependencyInjection/Compiler/TwigThemePass.php#L56 it could be
```suggestion
A theme is a set of directories to look for templates in. At application level, theme's templates are placed in a directory under `templates/themes` which has the same name as the theme.
```
What do you think of this attempt of being precise?